src/sparseml/pytorch/torchvision/train.py

```diff
@@ -466,6 +466,25 @@ def collate_fn(batch):
         return
 
     manager = ScheduledModifierManager.from_yaml(args.recipe_path)
+    if args.one_shot:
+        manager.apply(model)
+        checkpoint = {
```

Comment:
probably worth it to move the checkpoint building to a helper function since there will be repeated logic around what to save as well as recipe composition